tests/loaders/ndjson/test_bulk_export.py

```diff
@@ -240,7 +244,7 @@ async def test_happy_path(self):
     async def test_since_until(self):
         """Verify that we send since & until parameters correctly to the server"""
         self.mock_kickoff(
-            params="?_type=Condition%2CPatient&_since=2000-01-01T00%3A00%3A00%2B00.00&_until=2010",
+            params="?_type=Condition&_type=Patient&_since=2000-01-01T00%3A00%3A00%2B00.00&_until=2010",
```

Comment:
Generating a URL with duplicate _type fields vs a comma-separated field is an oddity of exploding the URL and putting it back together with urlparse. I could have manually fixed it after the fact, but the [spec](https://hl7.org/fhir/uv/bulkdata/export.html#bulk-data-kick-off-request) seems to prefer duplicate fields (even threatening that if folks don't love the comma approach, they'll take it out -- which seems highly unlikely at this adoption point, but still, I thought it was an interesting note.

Reply:
that is very unusual.